### PR TITLE
Create user-feedback-privacy.html

### PR DESCRIPTION
**Pull request by AuroroNightengale**

### CODE REVIEW

File: user-feedback-privacy.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <h2>Questions and Concerns.</h2>
+    <p>If you are concerned about the data stored <a href="support.html">Join the Quacky Support Server</a> or DM DuckMasterAl#0001 on Discord.</p>
+    <h2>How to Remove your data.</h2>
+    <p>If you would like to remove your data, you can <a href="support.html">Join the Quacky Support Server</a> and type "!new" in a commands channel to open a Ticket!</p>
+    <p>If you are banned from the support server, DM DuckMasterAl#0001 on Discord to remove the data.</p>
+</body>
+<footer>
+    <br/>
+    <h3><i>Note: We reserve the right to change this without notifying our users.</i></h3>
+    <h3>This policy was last updated August 25th, 2020.</h3>
```
Comment (DuckMasterAl) [inline user-feedback-privacy.html:40]: This should be updated to today's date.

File: user-feedback-privacy.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <div class="bottom2">
+        <div>
+            <picture>
+                <source type="image/webp" srcset="files/emojis/copyright.webp">
+                <source type="image/png" srcset="files/emojis/copyright.png">
+                <img src="files/emojis/copyright.png" class="cemoji" alt="©️" />
+            </picture> Copyright 2020 DuckMasterAl</div>
+        <div>This Website is Under the <a href="https://github.com/quacky-bot/quacky-bot.github.io/blob/master/LICENSE">GPL-3.0 License</a></div>
+    </div>
+</footer>
```
Comment (DuckMasterAl) [inline user-feedback-privacy.html:59]: This entire section does not currently exist in the code.

File: user-feedback-privacy.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Happy+Monkey&family=Londrina+Shadow&family=Montserrat&display=swap" rel="stylesheet">
+    <link href="files/styles/subpage.css" rel="stylesheet">
+    <link rel="icon" href="files/avatar.png">
+    <meta charset="UTF-8">
+    <meta name="description" content="Quacky is a moderation bot with lots of utility features. Quacky has custom moderation features like Voice and Text Channel Moderation to make life easier for your Moderators.">
+    <meta name="author" content="DuckMasterAl">
+    <meta name="keywords" content="discord bot, quacky, bot, quacky bot, quacky discord bot, discord, moderation bot discord, discord moderation bot, moderation bot">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Privacy Policy | Quacky Bot</title>
```
Comment (DuckMasterAl) [inline user-feedback-privacy.html:13]: The title should be updated.

**Review by DuckMasterAl — CHANGES_REQUESTED** [review]
Here are 3 things I noticed just from looking at the PR - I will do a more detailed review later.

File: user-feedback-privacy.html (hunk 3 above)
Comment (AuroroNightengale) [reply user-feedback-privacy.html]: Updated

File: user-feedback-privacy.html (hunk 2 above)
Comment (AuroroNightengale) [reply user-feedback-privacy.html]: Fixed. Just needs to be stylized!

File: user-feedback-privacy.html (hunk 1 above)
Comment (AuroroNightengale) [reply user-feedback-privacy.html]: Updated!

File: user-feedback-privacy.html (hunk 2 above)
Comment (DuckMasterAl) [reply user-feedback-privacy.html]: You didn't fix this, but I'll fix it when I make the style changes.

**Review by DuckMasterAl — APPROVED** [review]
Everything has been resolved in 510fadc, and this is now ready to be merged!

**Pull request merged**